src/librekey/key_store_g10.c

```diff
@@ -35,6 +35,13 @@
 #include "key_store_g10.h"
 
 #include "bn.h"
+#include "s2k.h"
+#include "symmetric.h"
+#include "readerwriter.h"
+
+#define G10_CBC_IV_SIZE 16
```

Comment:
G10 is basically being used as the file format name here I believe, so it does kind of make sense in the name IMO.